src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -177,6 +184,11 @@ bool IsValidTriggerCharacterForDocument(Document document, char triggerCharacter
 
                 if (currentRuleCache.TryGetValue(commitCharacterRules, out var currentRules))
                 {
+                    if (currentRules.IsEmpty)
+                    {
+                        return null;
+                    }
```

Comment:
doc?

Reply:
What?

Reply:
can you add docs here to explain why this is being done.